iati/home/models.py

```diff
@@ -117,12 +118,13 @@ def get_context(self, request, *args, **kwargs):
 
     def clean(self):
         """Override clean to remove trailing dashes from slugs with whitespaces."""
-        for lang in ['en', 'fr', 'es', 'pt']:
+        for lang in tuple(x[0] for x in settings.LANGUAGES):
```

Comment:
Using the languages setting as opposed to declaring inline - only `LANGUAGES` needs to be updated, then